web-ui/src/pages/PulseReportPage.jsx

```diff
@@ -194,73 +248,67 @@ const PulseReportPage = () => {
     //      because this permission has not been implemented yet.
     // if (selectHasPulseReportPermission(state)) loadPulses();
     loadPulses();
-    setTeamMembers(state.memberProfiles);
   }, [csrf, dateFrom, dateTo]);
 
-  const handleDateFromChange = dayJsDate => {
-    const date = new Date(dayJsDate.valueOf());
-    setDateFrom(date);
-    if (date > dateTo) setDateTo(date);
-  };
-
-  const handleDateToChange = dayJsDate => {
-    const date = new Date(dayJsDate.valueOf());
-    if (date < dateFrom) setDateFrom(date);
-    setDateTo(date);
-  };
-
-  const handleTeamMembersChange = members => {
-    setTeamMembers(members);
-  };
-
-  const responseSummary = () => {
-    let filteredPulses = pulses;
-    const teamMemberIds = teamMembers.map(member => member.id);
-    if (teamMemberIds.length) {
-      filteredPulses = pulses.filter(pulse =>
-        teamMemberIds.includes(pulse.teamMemberId))
-    }
+  useEffect(() => {
+    setTeamMembers(state.memberProfiles || []);
+  }, [csrf, state]);
 
+  const averageRow = scores => {
```

Comment:
This returns a single table row to be rendered in the tables of highest and lowest average scores.